### PR TITLE
add typos checker to pre-commit; bring back sqlfluff

### PR DESCRIPTION
**Pull request by atvaccaro**
# Description

Inspired by a typo found during the demo today.

Interestingly, it's not picking up on `agenices`.

## Type of change

- [ ] Bug fix (non-breaking change which fixes an issue)
- [x] New feature
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [ ] Documentation
- [ ] agencies.yml

## How has this been tested?
CI must pass.

### CODE REVIEW

File: airflow/dags/unzip_and_validate_gtfs_schedule/unzip_gtfs_schedule.py [hunk 1]
```diff
@@ -117,7 +117,7 @@ def process_feed_files(
     md5hash = hashlib.md5()
     if not is_valid:
         raise ValueError(
-            "Unparseable zip: File/directory structure within zipfile cannot be unpacked"
+            "Unparsable zip: File/directory structure within zipfile cannot be unpacked"
```
Comment (lauriemerrell) [inline airflow/dags/unzip_and_validate_gtfs_schedule/unzip_gtfs_schedule.py:120]: I feel a bit iffy about changing this spelling in an error where people may try to do string matching to query. Wiktionary lists it as an alternate spelling: https://en.wiktionary.org/wiki/parseable

File: ci/steps/release-helm-changes.sh [hunk 2]
```diff
@@ -24,7 +24,7 @@ IFS=: values_relpaths=( $RELEASE_HELM_VALUES )
 IFS=$' \t\n'
 
 if [[ ! -e $chart_path ]]; then
-  printf 'error: chart not found: %s\n' "$RELASE_HELM_CHART" >&2
+  printf 'error: chart not found: %s\n' "$RELEASE_HELM_CHART" >&2
```
Comment (lauriemerrell) [inline ci/steps/release-helm-changes.sh:27]: Are we sure the variable name is spelled correctly wherever it's set?

File: docs/analytics_new_analysts/data-management.md [hunk 3]
```diff
@@ -80,7 +80,7 @@ writer.save()
...
+Parquet is an "open source columnar storage format for use in data analysis systems." Columnar storage is more efficient as it is  easily compressed and the data is more homogeneous. CSV files utilize a row-based storage format which is harder to compress, a reason why Parquets files are preferable for larger datasets. Parquet files are faster to read than CSVs, as they have a higher querying speed and preserve datatypes (i.e. Number,  Timestamps, Points). They are best for intermediate data storage and large datasets (1GB+) on most any on-disk storage. This file format is also good for passing dataframes between Python and R. A similar option is [feather](https://blog.rstudio.com/2016/03/29/feather/).
```
Comment (lauriemerrell) [inline docs/analytics_new_analysts/data-management.md:83]: This is an accepted variant: https://grammarist.com/usage/homogenous-homogeneous/

**Review by lauriemerrell — COMMENTED** [review]
I don't agree with some of these "corrections", how can you override? Also when is this going to run? It makes me nervous that it runs on code files and just changes stuff, like the column name typo in funding programs-- if that were an incremental table fixing that would take more effort

**Comment by atvaccaro** [comment]
Going to close this PR and make a new one with tighter scope.

**Pull request closed without merging**